packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.html

```diff
@@ -3,7 +3,7 @@
     <lightning-card title={label}>
       <ul class="slds-has-dividers_around-space slds-m-vertical_medium">
         <template for:each={values} for:item="v">
-          <c-facet-value onselectvalue={onSelect} key={v.value} item={v}></c-facet-value>
+          <c-quantic-facet-value onselectvalue={onSelect} key={v.value} item={v}></c-quantic-facet-value>
```

Comment:
Is the `c-` prefix still needed now that there is `quantic` in the name?

Reply:
Yes the component tag must contain the namespace and `c-` (the default namespace) is therefore required.